Text to Speech/ViewController.swift

```diff
@@ -84,21 +81,31 @@ class ViewController: UIViewController {
     }
     
     func loadVoices() {
-        let failure = { (error: Error) in print(error) }
-        textToSpeech.getVoices(failure: failure) { voices in
+        textToSpeech.listVoices { [weak self] (response, error) in
```

Comment:
No need for `[weak self]`. The closure doesn't get captured by the `textToSpeech` instance (it's only a parameter), so there wouldn't be a strong reference cycle. I think it's better practice to keep `self` retained until the code below finishes, so that the app is guaranteed to get the new voices data.